articles/language/statements.md

```diff
@@ -233,7 +233,7 @@ Correspondingly, update-and-reassign statements exist for named items in user-de
 ```qsharp
 newtype Complex = (Re : Double, Im : Double);
 
-function ElementwisePlus(reals : Double[], ims : Double[]) : Complex[] {
+function ElementwisePlus(reals : Double[], ims : Double[]) : Complex {
```

Comment:
Ah crap, I missed in previous reviews that this was a summation, sorry. Maybe this would help?
```suggestion
function ComplexSum(reals : Double[], ims : Double[]) : Complex {
```

Reply:
@cgranade That's definitely a more fitting (not to mention elegant-sounding) name. Missed the merge on this PR but making a quick new one just for that.